tests/testthat/test-ssn.R

```diff
@@ -0,0 +1,24 @@
+context("SSNProvider works")
+
+test_that("SSNProvider works", {
+  aa <- SSNProvider$new()
+
+  expect_is(aa, "SSNProvider")
+  expect_is(aa, "R6")
+
+  expect_is(aa$render, "function")
+  expect_is(aa$render(), "character")
+  expect_match(aa$render(), "\\d\\d\\d-\\d\\d-\\d\\d\\d\\d") # :(
```

Comment:
Can you remove these comments?
```suggestion
  expect_match(aa$render(), "\\d\\d\\d-\\d\\d-\\d\\d\\d\\d")
```